docs/source/daml/intro/3_Data.rst

```diff
@@ -49,19 +49,19 @@ You have already encountered a few native DAML types: ``Party`` in :doc:`1_Token
 - ``RelTime``
   Stores a difference in time.
 
-The below scenario instantiates each one of these types, manipulates it where appropriate, and tests the result.
+The below script instantiates each one of these types, manipulates it where appropriate, and tests the result.
 
 .. literalinclude:: daml/daml-intro-3/Native.daml
   :language: daml
   :start-after: -- NATIVE_TEST_BEGIN
   :end-before: -- NATIVE_TEST_END
 
-Despite its simplicity, there are quite a few things to note in this scenario:
+Despite its simplicity, there are quite a few things to note in this script:
 
-- The ``import`` statements at the top import two packages from the DAML Standard Library, which contain all the date and time related functions we use here. More on packages, imports and the standard library later.
+- The ``import`` statements at the top import two packages from the DAML Standard Library, which contain all the date and time related functions we use here as well as the functions used in DAML Scripts. More on packages, imports and the standard library later.
 - Most of the variables are declared inside a ``let`` block.
 
-  That's because the ``scenario do`` block expects scenario actions like ``submit`` or ``getParty``. An integer like ``123`` is not an action, it's a pure expression, something we can evaluate without any ledger. You can think of the ``let`` as turning variable declaration into an action.
+  That's because the ``script do`` block expects script actions like ``submit`` or ``Party``. An integer like ``123`` is not an action, it's a pure expression, something we can evaluate without any ledger. You can think of the ``let`` as turning variable declaration into an action.
```

Comment:
Do we have a function `script : Script a -> Script a`?

Reply:
Yes, see comment above.